.github/workflows/ci_ubuntu_22.04_clang.yaml

```diff
@@ -0,0 +1,20 @@
+# Run Continuous Integration for the latest Ubuntu release
+# This mainly checks for issues/regressions in the native build
+name: native_compatibility_ubuntu22.04_clang
+on:
+  schedule:
+    - cron: '0 3 * * 0'
+  push:
+    branches:
+      - 'release-*'
+      - fix_native_compatibility_ci
+      - feature/clang_build
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run tests on Ubuntu 22.04 with Clang
+      run: docker run -v "${GITHUB_WORKSPACE}:/workspace" --env LC_ALL=C.UTF-8 --env LANG=C.UTF-8 --env DEBIAN_FRONTEND=noninteractive ubuntu:22.04 /bin/bash -c 'apt-get update && apt-get -y dist-upgrade && apt-get -y install build-essential clang cmake gfortran git valgrind libopenmpi-dev pkg-config python3 python3-pip python3-venv curl && apt-get -y remove libssl-dev && useradd -m -d /home/muscle3 muscle3 && su muscle3 -c -- "cp -r --preserve=mode /workspace /home/muscle3/muscle3" && su muscle3 -c -- "pip3 install -U pip setuptools wheel" && su muscle3 -c -- "pip3 install \"ymmsl>=0.12.0,<0.13\" qcg-pilotjob==0.13.1" && su muscle3 -c -- "cd /home/muscle3/muscle3 && CXXFLAGS=-fPIE OMPI_CXX=clang++ CXX=clang++ make test_examples"'
```

Comment:
`make test_examples` does not run `tox`, is that deliberate?

Reply:
I had to think about it, but I think yes. We already have the always-running CI, which does tox with python3.6, and a series of python compatibility runs. These tests run once a week (because they're relatively expensive) and test whether the native side builds correctly in a range of environments. So running the Python stuff again isn't really needed I think, although it wouldn't be much effort either to do it.